[storage][STG73] prepare for release (#11006)
Azure/azure-sdk-for-js: sdk/storage/storage-file-datalake/CHANGELOG.md

# Release History

## 12.1.0 (2020-09-08)

- Fixed `DataLakeFileClient.uploadStream()` to support `chunkSize` larger than `buffer.constants.MAX_LENGTH`.

> - Bug fix - `newPipeline()` now permits not specifying the credential. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628).

Reviewer: ```suggestion
- Bug fix - `credential` parameter of `newPipeline()` function is now optional. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628).
```

- Supported quick query. Added a new API `DataLakeFileClient.query()`.

> 

Reviewer: Quick Query?
